spec/features/job_seekers_can_search_for_vacancies_with_algolia_spec.rb

```diff
@@ -0,0 +1,334 @@
+        select I18n.t('jobs.sort_by.publish_on.descending')
+        click_on I18n.t('jobs.sort_by.submit')
+      end
+
+      expect(page.find('.vacancy:nth-child(1)')).to have_content(@first_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(2)')).to have_content(@second_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(3)')).to have_content(@third_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(4)')).to have_content(@fourth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(5)')).to have_content(@fifth_vacancy.job_title)
+    end
+
+    scenario 'oldest listing first', js: false, algolia: true do
+      visit jobs_path
+
+      within '.sortable-links' do
+        select I18n.t('jobs.sort_by.publish_on.ascending')
+        click_on I18n.t('jobs.sort_by.submit')
+      end
+
+      expect(page.find('.vacancy:nth-child(1)')).to have_content(@fifth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(2)')).to have_content(@fourth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(3)')).to have_content(@third_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(4)')).to have_content(@second_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(5)')).to have_content(@first_vacancy.job_title)
+    end
+
+    scenario 'least time to apply listing first', js: false, algolia: true do
+      visit jobs_path
+
+      within '.sortable-links' do
+        select I18n.t('jobs.sort_by.expiry_time.descending')
+        click_on I18n.t('jobs.sort_by.submit')
+      end
+
+      expect(page.find('.vacancy:nth-child(1)')).to have_content(@third_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(2)')).to have_content(@fifth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(3)')).to have_content(@second_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(4)')).to have_content(@first_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(5)')).to have_content(@fourth_vacancy.job_title)
+    end
+
+    scenario 'most time to apply listing first', js: false, algolia: true do
+      visit jobs_path
+
+      within '.sortable-links' do
+        select I18n.t('jobs.sort_by.expiry_time.ascending')
+        click_on I18n.t('jobs.sort_by.submit')
+      end
+
+      expect(page.find('.vacancy:nth-child(1)')).to have_content(@fourth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(2)')).to have_content(@first_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(3)')).to have_content(@second_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(4)')).to have_content(@fifth_vacancy.job_title)
+      expect(page.find('.vacancy:nth-child(5)')).to have_content(@third_vacancy.job_title)
+    end
+  end
+
+  context 'jobseekers can navigate between pages' do
+    let(:jobs_per_page) { 5 }
+
+    before do
+      VacancyAlgoliaSearchBuilder.class_variable_set(:@@default_hits_per_page, jobs_per_page)
```

Comment:
Is there an easier way to do this? I want to change the hits_per_page variable in the `VacancyAlgoliaSearchBuilder` from the test in order to paginate with a smaller number of vacancies